search/search-client-connect/src/test/java/io/camunda/search/connect/os/SearchDBExtension.java

```diff
@@ -26,9 +39,20 @@ public static SearchDBExtension create() {
     }
   }
 
+  /**
+   * @return {@link OpenSearchClient} client that is configured against container or AWS OS instance
+   *     depending on which extension was selected for execution.
```

Comment:
I feel it doesn't matter here on the API level, whether it is is container based or AWS OS instance. I would remove this